client/app/assets/less/redash/ant.less

```diff
@@ -36,3 +42,35 @@
 .ant-dropdown-in-bootstrap-modal {
   z-index: 1050;
 }
+
+// Make Ant labels more Redash like
+.ant-form-item-label label {
+  color: inherit;
+  white-space: initial;
```

Comment:
What's this for?

Reply:
To avoid these cuts with long labels (happens in their docs site if you inspect and change a form):
![vertical-cut](https://user-images.githubusercontent.com/3356951/50728505-fdb31900-1111-11e9-9dde-6dcb1dd34c00.png)

Reply:
Plz add a comment to remove this line when [this PR](https://github.com/ant-design/ant-design/pull/14135) lands.

Reply:
The change made it to [antd@3.12.0](https://github.com/ant-design/ant-design/releases/tag/3.12.0). Consider upgrading ⬆️💯

Reply:
I upgraded locally to `3.12.1` and it didn't work. Also, I didn't see your PR mentioned in the changelog, so I guess it hasn't been released yet :confused:

Reply:
> and it didn't work

Do you mean the upgrade didn't work? Or the `white-space` fix?

Reply:
> It should be there. Here's the real changelog ant-design/ant-design@3.12.0...master. I'll take a look and see why it's not applying.

At first I thought that as well. However, this is the exactly opposite, this is a comparison between 3.12.0 and master, so all the stuff on that link are not in this version.

Reply:
Ok ok I'm in! [3.12.2](https://github.com/ant-design/ant-design/releases/tag/3.12.2)

Reply:
haha, nice! :grin: